src/EditorFeatures/Core/InlineRename/IEditorInlineRenameService.cs

```diff
@@ -239,10 +239,7 @@ internal interface IInlineRenameInfo
         /// <see langword="true"/> if this operation succeeded, or <see langword="false"/> if it failed.
         /// </summary>
         bool TryOnAfterGlobalSymbolRenamed(Workspace workspace, IEnumerable<DocumentId> changedDocumentIDs, string replacementText);
-    }
 
-    internal interface IInlineRenameInfoWithFileRename : IInlineRenameInfo
```

Comment:
We have like 10 types defined in this file. Could you please move each type into their own?

Reply:
Yup. I can do this next!